build.gradle

```diff
@@ -76,7 +76,7 @@ sharedLibrary {
         dependency('org.jenkins-ci.plugins.workflow', 'workflow-multibranch', '2.26.1')
         dependency('org.jenkins-ci.plugins', 'pipeline-input-step', '456.vd8a_957db_5b_e9') // https://repo.jenkins-ci.org/public/org/jenkins-ci/plugins/pipeline-input-step/
         dependency('org.jenkins-ci.plugins', 'script-security', '1229.v4880b_b_e905a_6')
-        dependency('org.jenkins-ci.plugins', 'credentials', '1112.vc87b_7a_3597f6')
+        dependency('org.jenkins-ci.plugins', 'credentials', '1381.v2c3a_12074da_b_')
```

Comment:
Is it normal for such as big bump?
Also shall we keep the same version of this as the one on jenkins docker image?
cc: @gaiksaya  Thanks.

Reply:
Would recommend removing the dependency from here. I believe it was added to resolve CVE.

Reply:
Just removed this dependency from the file. Please take another look. Thanks!